compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/Types.java

```diff
@@ -4278,7 +4278,7 @@ private boolean equalityIntersectionExists(Set<BType> lhsTypes, Set<BType> rhsTy
         return matchFound;
     }
 
-    boolean validNumericStringOrXmlTypeExists(BType type, ValidateType validateType) {
+    boolean validationFunction(BType type, ValidateType validateType) {
```

Comment:
Meant using `ValidationFunction` for `ValidateType`, not `validNumericStringOrXmlTypeExists`.

On further though it should probably be `TypeExistenceValidationFunction`.